frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx

```diff
@@ -19,7 +21,8 @@ class PopinFixedHeader extends React.Component {
     super(props)
     this.state = {
       editTitle: false,
-      editTitleValue: props.rawTitle
+      editTitleValue: props.rawTitle,
+      popoverSpaceTitleOpen: false
```

Comment:
Wording suggestion, because the component is not used only for Spaces, if you apply, please change all document.

Reply:
```suggestion
      popoverTitleOpen: false
```